src/reducers/projectsReducer.js

```diff
@@ -1,16 +1,14 @@
 export const projectsReducer = (state = [], action) => {
-  switch(action.type) {
+  switch (action.type) {
     case 'GET_PROJECTS_SUCCESS':
       return action.projects
     case 'ADD_PROJECT_SUCCESS':
       return [...state, action.project]
     case 'DELETE_PROJECT_SUCCESS':
-      return state.filter(project => project.id !== action.projectID)
+      const message = action.projectID.split(' ')
```

Comment:
I think we should consider refactoring our BE to return just an ID in the response so we don't have to do as much cleaning here.